src/Internal/QueryM/Logging.purs

```diff
@@ -31,8 +31,8 @@ setupLogs logLevel customLogger = do
   logsRef <- liftEffect $ Ref.new Nil
   let
     -- Logger that stores a message in the queue
-    addLogEntry :: Message -> Effect Unit
-    addLogEntry msg = do
+    addLogEntry :: LogLevel -> Message -> Effect Unit
+    addLogEntry _ msg = do
```

Comment:
I'm unclear if we should filter with the logLevel here? or do we add messages in any case?

Reply:
@pb99u069 filter them.